packages/core/src/components/ModalNew/layouts/ModalMedia.module.scss

```diff
@@ -0,0 +1,8 @@
+.media {
```

Comment:
What if it's an image? Will users need to adjust the size?

Reply:
if they need a specific change they can pass `className`, but it should behave well with either image/lottie/video

Reply:
I feel like the way it's being provided now promises more than that 😅

Reply:
got ya, will check with different scenarios